x/profiles/types/chain_link.go

```diff
@@ -0,0 +1,176 @@
+package types
```

Comment:
Can you add the documentation to all the methods and functions that do not have it yet please?